test/cache.spec.js

```diff
@@ -1,11 +1,15 @@
-const assert = require('assert')
+import assert from 'assert'
+import Storage from 'orbit-db-storage-adapter'
+import Cache from '../src/Cache.js'
 
-const Storage = require('orbit-db-storage-adapter')
-const Cache = require('../src/Cache')
-const implementations = require('orbit-db-storage-adapter/test/implementations')
 const timeout = 50000
 
-implementations.forEach(implementation => {
+it('dummy', async () => {
```

Comment:
Do these empty tests pass?

Reply:
yes. is required to get browser tests to run.